lib/assets/javascripts/utils/paginable.js

```diff
@@ -4,14 +4,16 @@ export const paginableSelector = '.paginable';
 
 $(() => {
   const onAjaxSuccessHandler = (e, data) => {
-    $(e.target).closest(paginableSelector).html($(data));
+    $(e.target).closest(paginableSelector).replaceWith($(data));
```

Comment:
@briri @xsrust Can you double check this change for me?

The behaviour I saw was that each click on a sort link added a new `.paginable` DOM tree inside the `.paginable` one. I opted for `replaceWith` to fix that, and then moved the event listener to the 'body' element to catch click events on objects just added to the page.

Reply:
If it was nesting them then this should correct the issue and keep the DOM tree clean.
Delegating the listener to the body should work as well. We'll have to test all of the various scenarios to make sure the listeners still work as the table reloads (e.g. sorting after search, the view all link, etc.)